docs/create-pages.mdx

```diff
@@ -19,23 +19,65 @@ import { createPages } from 'waku';
 import { RootLayout } from './templates/root-layout';
 import { HomePage } from './templates/home-page';
 
-export default createPages(async ({ createPage, createLayout }) => {
+const router = createPages(async ({ createPage, createLayout }) => [
```

Comment:
I noticed the confusion starts here. We probably shouldn't call the result of `createPages` just a "router". We should call it pagesRouter or just pages.

Reply:
i like just `pages` 👍